address;source_location;insn;indent
0x1089b0;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1757;PUSH	%RBP;
0x1089b1;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1757;PUSH	%RBX;
0x1089b2;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1757;SUB	$0x18,%RSP;
0x1089b6;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1757;MOV	%FS:0x28,%RAX;
0x1089bf;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1757;MOV	%RAX,0x8(%RSP);
0x1089c4;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1757;XOR	%EAX,%EAX;
0x1089c6;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1758;CMPL	$0,0x27680f(%RIP);
0x1089cd;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1758;MOV	$-0x2,%EAX;
0x1089d2;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1758;JE	108a02 <PyTraceMalloc_Untrack+0x52>;
0x1089d4;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1758;MOV	%EDI,%EBX;
0x1089d6;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1763;MOV	0x2b2233(%RIP),%RDI;
0x1089dd;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1763;MOV	%RSI,%RBP;
0x1089e0;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1763;MOV	$0x1,%ESI;
0x1089e5;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1763;CALL	124de0 <PyThread_acquire_lock>;
0x1089ea;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1764;MOV	%RBP,%RSI;
0x1089ed;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1764;MOV	%EBX,%EDI;
0x1089ef;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1764;CALL	e671c <tracemalloc_remove_trace>;
0x1089f4;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1765;MOV	0x2b2215(%RIP),%RDI;
0x1089fb;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1765;CALL	124fd0 <PyThread_release_lock>;
0x108a00;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1767;XOR	%EAX,%EAX;
0x108a02;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;MOV	0x8(%RSP),%RDX;
0x108a07;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;SUB	%FS:0x28,%RDX;
0x108a10;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;JE	108a18 <PyTraceMalloc_Untrack+0x68>;
0x108a12;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;CALLQ	0x275d50(%RIP);
0x108a18;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;ADD	$0x18,%RSP;
0x108a1c;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;POP	%RBX;
0x108a1d;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;POP	%RBP;
0x108a1e;/usr/local/src/conda/python-3.9.19/Modules/_tracemalloc.c:1768;RET;
